changelogs/changelog.yaml

```diff
@@ -1 +1,11 @@
-releases: {}
+ancestor: null
+releases:
+  0.1.0:
+    changes:
+      release_summary: 'Our first release matches the ``hashi_vault`` lookup functionality
+        provided by ``community.general`` ver ``1.3.0``.
+
+'
```

Comment:
@felixfontein do you know where that odd extra whitespace comes from ? I've noticed that in other relnotes as well

Reply:
That's a YAML representation of https://github.com/ansible-collections/community.hashi_vault/pull/11/commits/1f53ba788f9ecbdc9799568133538d1a4455eb00#diff-564fa06adc205ddea56f8b8154bdb228ee47a7904bb469f56ec47f17a43f83a7

Reply:
Ah so extra whitespace because of the pipe ``|`` ? Would ``>-`` work better for example ?

Reply:
It depends on whether you plan to have multiple lines or even paragraphs. If you use `>` instead of `|`, it will be really hard to create new paragraphs since you need two newlines in the parsed string, and a double newline in YAML only gives you one newline in the parsed string. Probably `|-` would be better; I've adjusted https://github.com/ansible/community/wiki/ReleasingCollections accordingly.

Reply:
Should I adjust or remove the extra generated whitespace or just leave it?